<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BeamCentre</class>
 <widget class="QWidget" name="BeamCentre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Beam Centre Finder</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The centre positions of both the high angle and low angle detector banks. The values here if they exist will be used in any reduction initiated from the runs tab.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="title">
         <string>Centre Position</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="3" column="0">
          <widget class="QLineEdit" name="hab_pos_1_line_edit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lab_pos_1_line_edit"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lab_centre_label">
           <property name="text">
            <string>Centre Position LAB</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="hab_pos_2_line_edit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Detector</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="hab_centre_label">
           <property name="text">
            <string>Centre Position HAB</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lab_pos_2_line_edit"/>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="component_combo_box"/>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="update_lab_check_box">
           <property name="text">
            <string>Update LAB</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="update_hab_check_box">
           <property name="text">
            <string>Update HAB</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" rowspan="3">
       <widget class="QWidget" name="log_widget" native="true">
        <property name="minimumSize">
         <size>
          <width>491</width>
          <height>371</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="7" column="0" colspan="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Max iterations</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLineEdit" name="tolerance_line_edit">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is the minimum step size below which the algorithm will decide it has converged.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLineEdit" name="max_iterations_line_edit">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The maximum number of iterations the algorithm will perform.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QLineEdit" name="q_max_line_edit"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Tolerance</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QLabel" name="Q_to">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>from</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="Q_from">
           <property name="text">
            <string>from</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLineEdit" name="r_max_line_edit"/>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="Q_limits">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The Q range which will be considered when finding the beam centre.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Q Limits</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_8">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The radius limits which will be considered when finding the beam centre.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Radius Limits(mm)</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="q_min_line_edit"/>
         </item>
         <item row="11" column="0" colspan="5">
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>General Options</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="verbose_check_box">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If true the reduced quartile workspaces from each iteration will be output to the ADS.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Verbose</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="COM_check_box">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Option which controls whether the initial position is taken as the current centre position or determined from a centre of mass analysis of the scatter workspace.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Initial COM</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="r_min_line_edit"/>
         </item>
         <item row="13" column="0" colspan="5">
          <widget class="QGroupBox" name="groupBox">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Controls in which directions the algorithm will search for the beam centre.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="title">
            <string>Direction</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="left_right_check_box">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Left/Right</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="up_down_check_box">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Up/Down</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Run</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>146</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="run_button">
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>146</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lab_pos_1_line_edit</tabstop>
  <tabstop>lab_pos_2_line_edit</tabstop>
  <tabstop>hab_pos_1_line_edit</tabstop>
  <tabstop>hab_pos_2_line_edit</tabstop>
  <tabstop>component_combo_box</tabstop>
  <tabstop>update_lab_check_box</tabstop>
  <tabstop>update_hab_check_box</tabstop>
  <tabstop>r_min_line_edit</tabstop>
  <tabstop>r_max_line_edit</tabstop>
  <tabstop>q_min_line_edit</tabstop>
  <tabstop>q_max_line_edit</tabstop>
  <tabstop>max_iterations_line_edit</tabstop>
  <tabstop>tolerance_line_edit</tabstop>
  <tabstop>verbose_check_box</tabstop>
  <tabstop>COM_check_box</tabstop>
  <tabstop>left_right_check_box</tabstop>
  <tabstop>up_down_check_box</tabstop>
  <tabstop>run_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
